src/systemtask/SystemTask.h

```diff
@@ -90,6 +90,10 @@ namespace Pinetime {
       void OnIdle();
       void OnDim();
 
+      bool IsSleepDisabled() {
+        return doNotGoToSleep;
+      }
+
```

Comment:
This should probably be moved to displayapp, but I think this would require a lot more rework, so it should be done another time.

Reply:
I have to admit that the way "go to sleep" was implemented is a bit... tricky (multiple messages are exchanged between SystemTask and DisplayApp). This is because we need to ensure that DisplayApp stops sending new frames to the display and go to sleep before SystemTask can put devices to sleep (including the display and  SPI bus).